lib/internal/test_runner/test.js

```diff
@@ -213,10 +213,14 @@ class Test extends AsyncResource {
       }
     }
 
-    const test = new Test({ fn, name, parent, ...options });
+    return { fn, name, parent, ...options, ...overrides };
+  }
+
+  createSubtest(...props) {
+    const test = new Test(this.#parseProps(...props));
```

Comment:
i think this needs to use ReflectApply, otherwise it's vulnerable to Array.prototype[Symbol.iterator] being hijacked?